package.json

```diff
@@ -47,15 +47,18 @@
     "dependents-filter:semistandard": "npm run --silent dependents-filter-one -- -i dependents-data/source/semistandard.ndjson -o dependents-data/semistandard-filtered.ndjson --min-downloads=1000 --target-version='>=16.0.0' --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/semistandard.json)",
     "dependents-filter:standard": "npm run --silent dependents-filter-one -- -i dependents-data/source/standard.ndjson -o dependents-data/standard-filtered.ndjson --min-downloads=75000 --target-version='>=17.0.0' --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/standard.json)",
     "dependents-filter:ts-standard": "npm run --silent dependents-filter-one -- -i dependents-data/source/ts-standard.ndjson -o dependents-data/ts-standard-filtered.ndjson --min-downloads=1000 --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/ts-standard.json)",
-    "dependents-filter": "run-p dependents-filter:*",
-    "dependents-refresh-one": "cd dependents-data/source/ && list-dependents refresh --field engines -n",
+    "dependents-filter-simplify": "for PROJECT in $(jq -r 'join(\" \")' dependents-data/projects.json); do jq -s '[.[].repositoryUrl | sub(\"https://github.com/\";\"\";\"i\")] | unique | sort' \"dependents-data/$PROJECT-filtered.ndjson\" > \"dependents-data/$PROJECT.json\"; done",
+    "dependents-filter": "run-p dependents-filter:* && run-s dependents-filter-simplify",
+    "dependents-refresh-one": "cd dependents-data/source/ && list-dependents refresh --precision 3 --field engines -n",
     "dependents-refresh": "run-s --print-label \"dependents-refresh-one -- {%}\" -- $(jq -r 'join(\" \")' dependents-data/projects.json)",
-    "dependents-update-one": "cd dependents-data/source/ && list-dependents list --field engines -n",
+    "dependents-update-one": "cd dependents-data/source/ && list-dependents list --precision 3 --field engines -n",
     "dependents-update:neostandard": "run-s --print-label \"dependents-update-one -- {@}\" -- neostandard --include-historic",
     "dependents-update:all": "run-s --print-label \"dependents-update-one -- {%}\" -- $(jq -r 'join(\" \")' dependents-data/projects.json)",
     "dependents-update": "run-s --silent dependents-update:*",
     "prepare": "husky",
     "prepublishOnly": "run-s build",
+    "sync-refresh": "run-s dependents-refresh dependents-filter",
+    "sync-update": "run-s dependents-update dependents-filter",
```

Comment:
`sync-update` will be added as a separate workflow in a follow up PR.

The difference is:

* `refresh` only refreshes the data – it removes entries that are no longer using the relevant module and it updates relevant data of the entries, such as download count, which in turn can lead to the entry matching different in the filter step (the filter step is the one that does sorting etc to only take the top X many entries by download that fulfills Y conditions)
* `update` pulls a new complete list of dependents from ecosyste.ms and as such goes further than `refresh` by also adding new items to the lists. Its a slower and more disruptive process.